history-of-the-quakers-v2/original/07-book-3-chapter-13.adoc

```diff
@@ -68,19 +67,28 @@ and that many in it should fall and be taken away.`"
 
 The next is a short admonitory caution from George Bishop, of Bristol, addressed
 
-"`To the King and both houses of parliament.
+[.embedded-content-document.letter]
+--
 
-"`Thus saith the Lord,
+[.letter-heading]
+To the King and both houses of parliament.
 
-"`Meddle not with my people, because of their conscience to me,
+[.salutation]
+Thus saith the Lord,
+
+Meddle not with my people, because of their conscience to me,
 and banish them not out of the nation because of their conscience;
 for if you do I will send my plagues upon you, and you shall know that I am the Lord.
 
-"`Written in obedience to the Lord by his servant,`"
+Written in obedience to the Lord by his servant,
+
+[.signed-section-signature]
+G+++.+++ Bishop.
```

Comment:
Wasn't sure if I should switch the [sss] and [sscc] but I thought it looked a little awkward the way it was.